src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -74,14 +75,14 @@ interface GalleryTabInfo {
 export class GalleryViewerComponent extends React.Component<GalleryViewerComponentProps, GalleryViewerComponentState> {
   public static readonly OfficialSamplesTitle = "Official samples";
   public static readonly PublicGalleryTitle = "Public gallery";
-  public static readonly FavoritesTitle = "Liked";
-  public static readonly PublishedTitle = "Your published work";
+  public static readonly FavoritesTitle = "My favorites";
+  public static readonly PublishedTitle = "My published work";
 
   private static readonly rowsPerPage = 5;
 
   private static readonly mostViewedText = "Most viewed";
   private static readonly mostDownloadedText = "Most downloaded";
-  private static readonly mostFavoritedText = "Most liked";
+  private static readonly mostFavoritedText = "Most favorited";
```

Comment:
>favorited [](start = 52, length = 9)

Guess if this is valid. Favorite is a noun and not a verb.

Reply:
Not a language expert but [google](https://www.google.com/search?ei=5h0UYL69I8DA3LUP4NejkAE&q=favorited&oq=favorited&gs_lcp=CgZwc3ktYWIQAzIKCAAQkQIQRhD5ATICCAAyBAgAEAoyBAgAEAoyBAgAEAoyAggAMgIIADIECAAQCjIECAAQCjICCAA6CgguELADEEMQkwI6BwgAELADEENQvvsCWL77AmCg_QJoAXACeACAAZUBiAGVAZIBAzAuMZgBAKABAaoBB2d3cy13aXrIAQrAAQE&sclient=psy-ab&ved=0ahUKEwi-yP6pr8HuAhVAILcAHeDrCBIQ4dUDCA0&uact=5) says that "favorited" is a word